tests/models/conf/config.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
hall it rather go to `tests/data/conf`? (not important)

Reply:
Simpler as hydra need relative path.

Reply:
There are multiple initialize methods you can use that might make more sense to your use case.
Take a look at the docs [here[(https://hydra.cc/docs/experimental/compose_api#initialization-methods).